### PR TITLE
chore: core: enforce revert style guide

### PR DESCRIPTION
**Pull request by davidprepo**
Spec: https://www.notion.so/prepo/SC-Backlog-Tasks-0c6725bb870f4c22b7ba771347333662#274ae93c1a6c4b5894d152c7e8997928

- Revert messages in older core contracts don’t follow new format described in [[Smart Contract Handbook](https://www.notion.so/Smart-Contract-Handbook-73d78c1d6eec46e2adb225d7162b48c4)](https://www.notion.so/Smart-Contract-Handbook-73d78c1d6eec46e2adb225d7162b48c4)

### CODE REVIEW

File: apps/smart-contracts/core/contracts/ManagerWithdrawHook.sol [hunk 1]
```diff
@@ -27,7 +27,7 @@ contract ManagerWithdrawHook is
   ) external view override {
     require(
       collateral.getReserve() - _amountAfterFee >= getMinReserve(),
-      "reserve would fall below minimum"
+      "Reserve - amountAfterFee < minReserve"
```
Comment (davidprepo) [inline apps/smart-contracts/core/contracts/ManagerWithdrawHook.sol:30]: Not sure on this one. The change fits the style guide, but the original message feels better to me.

Comment (XavierEkkel) [inline apps/smart-contracts/core/contracts/ManagerWithdrawHook.sol:30]:
```suggestion
      "Falls below min reserve"
```

File: apps/smart-contracts/core/test/WithdrawHook.test.ts [hunk 2]
```diff
@@ -365,7 +365,7 @@ describe('=> WithdrawHook', () => {
 
         await expect(
           withdrawHook.connect(collateralSigner).hook(user.address, 1, 1)
-        ).to.revertedWith('global withdraw limit exceeded')
+        ).to.revertedWith('Global withdraw limit exceeded')
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/test/WithdrawHook.test.ts:368]:
```suggestion
        ).revertedWith('Global withdraw limit exceeded')
```

File: apps/smart-contracts/core/test/ManagerWithdrawHook.test.ts [hunk 3]
```diff
@@ -341,7 +341,7 @@ describe('=> ManagerWithdrawHook', () => {
 
       await expect(
         managerWithdrawHook.connect(user).hook(user.address, IGNORED_ARGUMENT, amountToWithdraw)
-      ).to.revertedWith('reserve would fall below minimum')
+      ).to.revertedWith('Reserve - amountAfterFee < minReserve')
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/test/ManagerWithdrawHook.test.ts:344]:
```suggestion
      ).revertedWith('Falls below min reserve')
```

File: apps/smart-contracts/core/test/DepositHook.test.ts [hunk 4]
```diff
@@ -227,7 +227,7 @@ describe('=> DepositHook', () => {
         depositHook
           .connect(collateralSigner)
           .hook(user.address, TEST_AMOUNT_BEFORE_FEE, TEST_AMOUNT_AFTER_FEE)
-      ).to.revertedWith('depositor not allowed')
+      ).to.revertedWith('Depositor not allowed')
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/test/DepositHook.test.ts:230]:
```suggestion
      ).revertedWith('Depositor not allowed')
```

**Review by XavierEkkel — APPROVED** [review]
LGTM after nits

**Pull request merged**